tests/unit/unit_tests.h

```diff
@@ -5,9 +5,18 @@
  *  Define things in here that are going to be needed across most unit tests.
  */
 
+#include "splinterdb/public_platform.h"
+
 /* Name of SplinterDB device created for unit-tests */
-#define TEST_DB_NAME "unit_tests_db"
+#define TEST_DB_NAME "splinterdb_unit_tests_db"
```

Comment:
Slightly unrelated bug-fix ... arising from one of @rosenhouse 's suggestion on an earlier PR.